src/registrar/tests/test_models_domain.py

```diff
@@ -637,9 +929,11 @@ def test_user_cannot_add_non_subordinate_with_ip(self):
                 which is not a subdomain of the domain and has IP addresses
             Then Domain raises a user-friendly error
         """
-        raise
+        dotgovnameserver = "mynameserverdotgov.gov"
+
+        with self.assertRaises(ValueError):
```

Comment:
Seeing these `ValueError` asserts makes me wonder if we want to use subclasses of `ValueError` for the different particular errors that could be raised when we try this.